.github/workflows/ci.yml

```diff
@@ -14,14 +14,10 @@ jobs:
           cache: 'npm'
       - name: Install
         run: npm ci
-      - name: Format check
-        run: npm run format:check
       - name: Lint
         run: npm run lint
       - name: Build
         run: npm run build
-      - name: Test
-        run: npm test --coverage
```

Comment:
Those checks have been removed temporarily and it will be added again in a follow up PR https://github.com/neofinancial/ticket-check-action/pull/41